src/test/java/com/microfocus/application/automation/tools/results/service/TestDefaultExternalEntityUploadServiceImpl.java

```diff
@@ -22,11 +22,29 @@
 
 package com.microfocus.application.automation.tools.results.service;
 
+import hudson.model.AbstractBuild;
+import hudson.model.FreeStyleProject;
+//import org.junit.Before;
+//import org.junit.ClassRule;
+//import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import java.io.IOException;
 
 public class TestDefaultExternalEntityUploadServiceImpl {
 
-	
-	private static void testJunit(int i) throws Exception{
+	FreeStyleProject project;
+
+	//@ClassRule
```

Comment:
Do you have an idea why are all those commented out?

Reply:
I talked with the author of the test. He said this test can only be ran manually. Can't be ran as Junit test. Those tags are for Junit test. It will fail in Junit test. Actually, this test never be ran before.